src/lib/database/pocky-db.ts

```diff
@@ -54,7 +55,11 @@ export default class PockyDB implements PockyDbInterface {
 			return dbConstants.pegError;
 		}
 
-		if (!senderHasPegs) {
+		// TODO: make this faster and less repetitive by refactoring getting the list of penaltyKeywords.
+		const penaltyKeywords = this.config.getStringConfig('penaltyKeyword');
+
+		if (!senderHasPegs
+			&&!penaltyKeywords.some(keyword => comment.toLowerCase().includes(keyword.toLowerCase()))) {
```

Comment:
Should move this sort of logic to hasSparePegs. Also it doesn't work.

Reply:
Yeah will do.